app/models/specialist_document.rb

```diff
@@ -15,8 +15,8 @@ def initialize(content_store_response)
     @headers = headers_list(content_store_response.dig("details", "headers"))
   end
 
-  def facet_values
-    @facet_values ||= selected_facets.map do |selected_facet|
+  def assigned_facets
```

Comment:
I'm not clear on what the difference is here. What is an "assigned" facet? There isn't a reference to assigned facets in the content item (there's facets in links/finder, and there's - confusingly - metadata in the details). 

I'm keen that we don't introduce more cryptic terms in this already quite complex system.

Reply:
@KludgeKML 
I think mainly what caused confusion for me is the `values` in `facet_values` as it refers to metadata field values, but those in turn also have a values: key which then in turn contain `label` and `values` from the `allowed_values`. so in the presenter you get instances where `facet_values[:value].map { |value| value[:label] && value[:value]`. i get really lost as to which `value` is which. 

Originally refactored to `facets` so we can have `facets[:value].map { |facet_value| facet_value[:value] }` but Leena was concerned about `facets` not being a good name cause we have `finder.facets` which returns full list of facets, hence suggested `assigned_facets`. I'm happy to go for other suggestions.

Reply:
Perhaps `facets_with_metadata_values` is a better term for this, then?